tsconfig.json

```diff
@@ -22,7 +22,7 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "outDir": "dist",
-    "removeComments": true,
+    "removeComments": false,
```

Comment:
Main change, from this thread https://stackoverflow.com/questions/44580985/how-to-export-and-consume-jsdoc-comments-in-typescript-npm-packages